src/dbnode/client/connection_pool.go

```diff
@@ -31,7 +31,7 @@ import (
 
 	"github.com/m3db/m3/src/dbnode/generated/thrift/rpc"
 	"github.com/m3db/m3/src/dbnode/topology"
-	xclose "github.com/m3db/m3/src/x/close"
+	xresource "github.com/m3db/m3/src/x/resource"
```

Comment:
nit: why is this `xresource` here and `resource` elsewhere? No preference other than being consistent, unless we're import aliasing because of locally-scoped var names, in which case the vars should change instead.

Also, more generally, if we want to formally adopt import alias consistency for `x/` packages (i.e., "all imports of `x/...` must be aliased as `x<pkg>`), that'd be okay too. (Although we should make sure that we're not actually naming packages `x<pkg>`, just aliasing them that way, since package names should match directory names.)

Reply:
In this PR, I just followed the convention in the file (i.e. if `xclose` then `xresource` and vice-versa).

I think the other case for collisions (besides what you cited as colliding w/ a var name) is colliding w/ some other package of same name. I think this is a compelling reason for just always doing `xpackage` because then you can consistently do that and never collide w/ a package of same name.

I agree w/ your proposal in 2nd half of response, where we say we always do `xpackage` (based on reasoning above) and also ensure we never actually name the package w/ the `x` prefix. So I can change any existing refs in this PR that are not aliased to `xresource` to reflect this.

Reply:
Agree on all counts - sounds good!